airflow/models/dag.py

```diff
@@ -1748,7 +1749,7 @@ def create_dagrun(
         conf: dict | None = None,
         run_type: DagRunType | None = None,
         session: Session = NEW_SESSION,
-        dag_hash: str | None = None,
+        dag_version_id: str | None = None,
```

Comment:
Is this type hint correct? I thought the field would return a UUID object instead.